src/components/DraftImportCsvModal.vue

```diff
@@ -59,43 +54,44 @@ import {
   IonToolbar,
   modalController
 } from "@ionic/vue";
-import { defineComponent } from "vue";
+import { defineProps, ref } from "vue";
 import { closeOutline, saveOutline } from "ionicons/icons";
 import { translate } from '@/i18n'
+import { showToast } from "@/utils";
+
+
+const props = defineProps(["fileColumns", "content", "countId"])
+const productIdentifications = process.env.VUE_APP_PRDT_IDENT ? JSON.parse(process.env.VUE_APP_PRDT_IDENT) : []
+
+const selectedIdentifier = ref('')
+const selectedColumn = ref('')
+
+function closeModal(identifierData: any) {
```

Comment:
As we will always be passing a valid object when closing modal after successful csv upload, there is no need to check for identifierData length and its existence, simply provide it a default value of empty object and pass it to dismiss method.
```suggestion
function closeModal(identifierData = {}) {
```